android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -150,6 +151,7 @@ public class MwmActivity extends BaseMwmFragmentActivity
   public static final String EXTRA_TRACK_ID = "track_id";
   public static final String EXTRA_UPDATE_THEME = "update_theme";
   private static final String EXTRA_CONSUMED = "mwm.extra.intent.processed";
+  private boolean mDialogShown = false;
```

Comment:
This activity if very big, its not clear what is this dialog about without searching for usages.
Please rename to e.g. `mPreciseLocationDialogShown`.